latex/Research_Strategy.tex

```diff
@@ -356,10 +356,10 @@
 % Sustained funding will allow us to do that. 
 % (This will also help answer the argument that we don't need funding for this...)
 \textbf{This new experimental data is critical to maximize impact on the modeling community.} 
-While the community has already benefitted considerably from SAMPL, as indicated by the nearly 100 publications on SAMPL (see attached list), the citation count, and lessons learned, the benefits are not what they could have been if the effort had funding.
+While the community has already benefitted considerably from SAMPL, as indicated by the nearly 100 publications on SAMPL (see attached list), the citation count, and the growing participation (Figure~\ref{figure:sampl_history}), the benefits are not what they could have been if the effort had funding.
```

Comment:
> the benefits are not what they could have been if the effort had funding.

Maybe "progress could be accelerated greatly with a funded initiative of the kind we describe"?